adhocracy4/polls/static/PollDetail/PollQuestion.jsx

```diff
@@ -30,9 +30,10 @@ export const PollQuestion = (props) => {
   const maxlength = 250
 
   useEffect(() => {
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
added to avoid below error, need refactor.
`React Hook useEffect contains a call to 'setUserChoices'. Without a list of dependencies, this can lead to an infinite chain of updates. To fix this, pass [props.question.userChoices, props.errors] as a second argument to the useEffect Hook`